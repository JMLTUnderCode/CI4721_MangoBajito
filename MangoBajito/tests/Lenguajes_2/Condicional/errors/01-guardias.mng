# Test 01: Errores de guardias
se_prende(){
	si_es_asi (12) {}
	o_asi (1.23) {}
	o_asi (12.4e+230) {}
	o_asi ('c') {}
	o_asi ("cadena") {};
};