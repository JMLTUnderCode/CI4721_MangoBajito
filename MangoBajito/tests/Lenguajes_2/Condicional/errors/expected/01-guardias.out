| AcomÃ³date que te lo estan analizando d:-)
Error: TYPE_ERROR
  - Sendo peo en la linea 3, columna 18: Tas en droga?. Tienes CondiciÃ³n de tipo 'mango', se esperaba 'tas_claro'.
  - Sendo peo en la linea 4, columna 13: Tas en droga?. Tienes CondiciÃ³n de tipo 'manguita', se esperaba 'tas_claro'.
  - Sendo peo en la linea 5, columna 18: Tas en droga?. Tienes CondiciÃ³n de tipo 'manguangua', se esperaba 'tas_claro'.
  - Sendo peo en la linea 6, columna 12: Tas en droga?. Tienes CondiciÃ³n de tipo 'negro', se esperaba 'tas_claro'.
  - Sendo peo en la linea 7, columna 17: Tas en droga?. Tienes CondiciÃ³n de tipo 'higuerote', se esperaba 'tas_claro'.
[1;31m[5m
               =======================================================
                       ---->        Error Program        <----        
               =======================================================
[0m
| Hay un poquito de cÃ³digo en tu marihuana d:-]
